helm/nessie/README.md

```diff
@@ -41,6 +41,9 @@ The following table lists the configurable parameters of the Nessie chart and th
 | mongodb | Configuration specific to `versionStoreType: MONGO` | |
 | mongodb.name | The database name | `nessie` |
 | mongodb.connectionString | The database connection string | `mongodb://localhost:27017` |
+| mongodb.secret.name | The name of the secret where database credentials are stored | `mongodb-creds` |
+| mongodb.secret.username | The name of the username key inside the secret | `mongodb_username` |
+| mongodb.secret.password | The name of the password key inside the secret | `mongodb_password` |
```

Comment:
let's also please add a section further below about those mongo secrets (similar to how we do for other secrets)